### ISSUE TITLE
Allow assets to be links

### ISSUE THREAD

**Issue by janhohenheim** [bash/gilderoy#3]
```toml
[assets.pic]
source = "https://avatars2.githubusercontent.com/u/9047632"
```
This does not work at the moment, as gilderoy expects all assets to be local files.

**Comment by tautropfli** [comment]
What is the desired behaviour with external resources? Should gilderoy fetch them and keep a local copy?

The idea behind `assets` is that gilderoy processes them (e.g. sass) and appends a hash to the filename (for more efficient caching).

**Comment by janhohenheim** [comment]
Fair enough, good question. I think I opened this issue as an [X Y Problem](http://xyproblem.info/). Closed in favour of #4
